openassessment/xblock/data_conversion.py

```diff
@@ -113,9 +124,13 @@ def create_prompts_list(prompt_or_serialized_prompts):
     if prompt_or_serialized_prompts is None:
         prompt_or_serialized_prompts = ''
 
-    try:
-        prompts = json.loads(prompt_or_serialized_prompts)
-    except ValueError:
+    if type(prompt_or_serialized_prompts) is dict:
+        # Lazy translation objects cannot be parsed to json normally
+        # https://docs.djangoproject.com/en/1.8/topics/serialization/#serialization-formats-json
+        # we need to pass it with a custom encoder.
+        decode_promts = json.dumps(prompt_or_serialized_prompts, cls=LazyEncoder)
```

Comment:
Nit: I guess it's supposed to be `decode_prompts`, not `decode_promts`.
(If so, then please correct also the line below)